### PR TITLE
Added a few tests for atomic.py

### PR DESCRIPTION
**Pull request by vidit23**

### CODE REVIEW

**Comment by wkerzendorf** [comment]
@vidit23 that did it - can you close the other PR? thanks

File: tardis/tests/test_atomic.py [hunk 1]
```diff
@@ -15,6 +15,19 @@ def test_ionization_h5_readin():
     data = atomic.read_ionization_data(atomic.default_atom_h5_path)
     hi_ionization = data['ionization_energy'][0]
     testing.assert_almost_equal(hi_ionization, 13.59844, decimal=4)
+    
+def test_lines_h5_reading():
+    data=atomic.read_lines_data(atomic.default_atom_h5_path)
```
Comment (kaushik94) [inline tardis/tests/test_atomic.py]: I think the convension is 'readin'

**Comment by vidit23** [comment]
@wkerzendorf Should I close this PR?

**Comment by wkerzendorf** [comment]
remind me, why?

**Comment by vidit23** [comment]
I read off the other PR about the same test, that you will evaluate based on these requests.
I assumed there was no point in leaving them open until they are ready to be merged.

**Comment by wkerzendorf** [comment]
that's right. It's difficult to find problems for so many students, sorry about that. very rarely if a student is working on something that is needed right now will we merge, but it doesn't go into the evaluation if your PR is merged or not.

**Comment by vidit23** [comment]
@wkerzendorf I added a test for read_collision_data. Please have a look whenever possible

**Comment by wkerzendorf** [comment]
That looks good. Maybe try to rebase it on the current master as it does not seem mergeable. Now is also a good time to start thinking about your application. The more specific you are the better.

**Comment by kaushik94** [comment]
@vidit23  must be because of #225 . Just pull the current master into yours.

**Comment by vidit23** [comment]
@wkerzendorf I will start working on it.
@kaushik94 Thanks for the help. I had made the PR from the site itself and I have no idea how to update the fork. I was thinking of making a seperate PR with all tests included

**Comment by kaushik94** [comment]
@vidit23  Yes thats how you are supposed to do a PR. The problem you are facing here is that someone else also edited the same file that you are trying to, so the tardis master and your master are out of sync, so you would have to "pull" the changes. just run the following in your master as well as in your current branch.
```
git remote add origin https://github.com/tardis-sn/tardis
git pull origin/master
``` 
This will update your master and your current branch and then you can push your changes, don't have to open a PR. Hope it helps.

**Comment by wkerzendorf** [comment]
then you get merge commits - better to do a rebase ;-)

**Comment by vidit23** [comment]
@wkerzendorf @kaushik94
Thank you... Didn't read the comments before making another PR.
I'll definitely do this next time

**Pull request closed without merging**